### PR TITLE
chore: refactor carbonProject (cms) to cms

### PR DESCRIPTION
**Pull request by 0xemc**
## Description

The 'carbonProject' moniker was confusing when trying to associate where content and project data was coming from as there are many other references to `carbonProject` from other sources 

## Related Ticket
none

## Notes For QA

Should be no impact on behaviour

### CODE REVIEW

File: carbonmark-api/test/fixtures/cms.ts [hunk 1]
```diff
@@ -31,15 +40,13 @@ const carbonProject = {
...
 /** Fixtures for queries to the carbon-projects cms */
-const fixtures = {
-  projectContent,
-  project,
-  /** Project entry in `fetchAllProjects` query */
+export default {
+  cmsProject,
+  cmsProjectContent,
   carbonProject,
+  /** Project entry in `fetchAllProjects` query */
```
Comment (biwano) [inline carbonmark-api/test/fixtures/cms.ts:51]:
```suggestion
  /** Project entry in `fetchAllProjects` query */
  carbonProject,
```

File: carbonmark-api/test/routes/projects/[id]/get.test.ts [hunk 2]
```diff
@@ -45,7 +47,7 @@ describe("GET /projects/:id", () => {
       .post("", /.*getActivitiesByProjectId.*/i)
       .reply(200, {
         data: {
-          activities: fixtures.activities,
+          activities: fixtures.marketplace.activities,
```
Comment (biwano) [inline carbonmark-api/test/routes/projects/[id]/get.test.ts:50]: Why not use the same pattern as the one you used for mockCmsProject ?
ie add
 `const mockMarketplaceActivities = fixtures.marketplace.activities;`
and reuse that variable.

File: carbonmark-api/test/routes/projects/get.test.ts [hunk 3]
```diff
@@ -120,19 +132,18 @@ describe("GET /projects", () => {
...
       {
-        ...pick(digitalCarbon.digitalCarbonProject, ["region"]),
+        region: digitalCarbon.digitalCarbonProject.region,
         methodologies: [
           {
-            id: carbonProjects.carbonProject.methodologies[0].id,
-            category: carbonProjects.carbonProject.methodologies[0].category,
-            name: carbonProjects.carbonProject.methodologies[0].name,
+            id: mockCmsProject.methodologies?.[0]?.id,
+            category: mockCmsProject?.methodologies?.[0]?.category,
```
Comment (biwano) [inline carbonmark-api/test/routes/projects/get.test.ts:139]: nitpick: There is an inconsistent use of `?.` here (not used on the line above)

**Review by biwano — APPROVED** [review]
Cool

File: carbonmark-api/test/routes/projects/[id]/get.test.ts (hunk 2 above)
Comment (0xemc) [reply carbonmark-api/test/routes/projects/[id]/get.test.ts:50]: Good idea, done.

File: carbonmark-api/test/routes/projects/get.test.ts (hunk 3 above)
Comment (0xemc) [reply carbonmark-api/test/routes/projects/get.test.ts:139]: ok, cleaned up

**Pull request merged**